packages/slice-machine/components/DeleteDocumentsDrawer/index.tsx

```diff
@@ -1,3 +1,2 @@
 export { SoftDeleteDocumentsDrawer } from "./SoftDeleteDocumentsDrawer";
 export { HardDeleteDocumentsDrawer } from "./HardDeleteDocumentsDrawer";
-export { ReferencesErrorDrawer } from "./ReferencesErrorDrawer";
```

Comment:
This ReferencesErrorDrawer was not used at all anymore and since a long time (10 months at least), correct?

Reply:
Correct! It seems you left some related code inside `cypress/pages/changes/drawers`

Reply:
Yes, I can remove it, but I don't touch Cypress at all anymore usually as everything is skipped, it will be removed when doing the global removal of Cypress. That I'm planning to do super soon btw. WDYT?

Reply:
That works for me